packages/components/src/Actions/ActionFile/ActionFile.component.js

```diff
@@ -117,6 +117,7 @@ class ActionFile extends React.Component {
 			tooltipLabel,
 			label,
 			tooltipPlacement,
+			accept,
```

Comment:
we can accept all others props instead
```suggestion
			...rest,
		}
```

Reply:
no please don t do that. It generate lots of proptype warning.
```
React does not recognize the `iconTransform` prop on a DOM element
```